test/browser_tests/spec_suites/content/blog.js

```diff
@@ -134,4 +134,35 @@ describe( 'The Blog Page', function() {
   it( 'should include a page input with value set to 1', function() {
     expect( page.paginationPageInput.getAttribute( 'value' ) === 1 );
   } );
+
+  it( 'should set an initial height on filter expandable that does not cut off submit button when page loads with 5 filter options selected', function() {
+    // open filter and select 5 options
+    page.searchFilterShowBtn.click();
+    browser.sleep( 1000 );
```

Comment:
I vaguely remember discussing this topic with @KimberlyMunoz. Nearly all protractor methods return promises so this spec could be rewritten as:

```
it( 'should set an initial height on filter expandable that does not cut off submit button when page loads with 5 filter options selected', function() {

  // open filter and select 5 options
  page.searchFilterShowBtn.click().then( function() {
    page.multiSelectSearch.isDisplayed().then( function() {
      page.multiSelectSearch.click().then( function() {
        var options = browser.element.all( by.css( '.cf-multi-select_label' ) );
        for ( var i = 0; i < 5; i++ ) {
          options.get(i).click();
        }

        // 5 choices should appear
        var choices = browser.element.all( by.css( '.cf-multi-select_choices label' ) );
        expect( choices.count() ).toBe( 5 );

        // submit choices
        browser.executeScript( 'arguments[0].scrollIntoView();', page.searchFilterSubmitBtn.getWebElement() );
        page.searchFilterSubmitBtn.click().then( function() {
          // submit button should be visible in open filter expandable
          // after page is reloaded
          browser.getCurrentUrl().then( function( newUrl ) {
            browser.get( newUrl );
            expect( page.searchFilterSubmitBtn.isDisplayed() ).toBe( true );
          } );
        } );

      } );
    } );
  } );

} );
```

The downside is that it's a bit callback hell-y. The plus side is that it'll execute a lot faster.